### PR TITLE
add pysqlite3 recipe

### PR DESCRIPTION
**Pull request by JakaKokosar**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by JakaKokosar** [comment]
@conda-forge/staged-recipes

File: recipes/pysqlite3/run_test.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import pysqlite3.dbapi2 as sqlite3
```
Comment (jan-janssen) [inline recipes/pysqlite3/run_test.py]: Is this file actually used?

Comment (chrisburr) [reply recipes/pysqlite3/run_test.py]: Yeah, it should be ran automatically (the same way `build.sh` is for the build step).

Comment (jan-janssen) [reply recipes/pysqlite3/run_test.py]: Sorry - I did not knew this.

File: recipes/pysqlite3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: 74bfe92c4febce11ee05f741d9b6406e1919440807a315c65f2e0b489fe9eb75
+
+build:
+  number: 0
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (scopatz) [inline recipes/pysqlite3/meta.yaml]: it looks like it just needs a c compiler

File: recipes/pysqlite3/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  imports:
+    - pysqlite3
+    - pysqlite3.dbapi2
+
+about:
+  home: https://github.com/coleifer/pysqlite3
+  license: zlib
+  license_file: LICENSE
```
Comment (scopatz) [inline recipes/pysqlite3/meta.yaml]: please add license_family: other

**Comment by JakaKokosar** [comment]
@scopatz Is that OK? Didn't know you can add multiple sources, that is cool! Thanks!

**Comment by scopatz** [comment]
yes , that is totally OK and much preferred

**Comment by JakaKokosar** [comment]
@scopatz  Can this be merged?

**Comment by JakaKokosar** [comment]
Well I'm torn here. As far as I can tell I can't use FTS5 extension because its not enabled in that project (https://github.com/conda-forge/sqlite-feedstock/blob/master/recipe/build.sh#L11). I want to have pysqlite3 as a requirement in Orange3-Bioinformatis project which will enable me to use FTS5 extension.

**Comment by JakaKokosar** [comment]
https://github.com/conda-forge/sqlite-feedstock/pull/48

**Comment by jakirkham** [comment]
Appears to have been merged. Anything else needed before closing this?

**Comment by JakaKokosar** [comment]
I guess not. You can close. Thanks!

**Comment by jakirkham** [comment]
Thanks @JakaKokosar for raising and handling this!😀

**Pull request closed without merging**